Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -117,18 +117,24 @@ export function runQTest(args: QTestArguments): Result {
     qTestDirToDeploy = qTestDirToDeploy || args.qTestDirToDeploy;
 
     // Microsoft internal cloud service use only
-    let qTestContextInfo = Environment.hasVariable("[Sdk.BuildXL]qtestContextInfo") ? f`${Environment.getFileValue("[Sdk.BuildXL]qtestContextInfo")}` : undefined;
-    let untrackingCBPaths =  Environment.hasVariable("[Sdk.BuildXL]qtestContextInfo") ? {
-        unsafe: {
-            untrackedPaths: [
-                qTestContextInfo,
-            ],
-            untrackedScopes: [
-                d`d:/data`,
-                d`d:/app`,
-            ]
-        }
-    } : {};
+    let qTestContextInfoPath = undefined;
+    let untrackingCBPaths = {};
+    if (Environment.hasVariable("[Sdk.BuildXL]qtestContextInfo")){
+        const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL]qtestContextInfo");
+        qTestContextInfoPath = qTestContextInfoFile.path;
+        untrackingCBPaths =  {
+            unsafe: {
+                untrackedPaths: [
+                    qTestContextInfoFile,
+                ],
+                untrackedScopes: [
+                    d`d:/data`,
+                    d`d:/app`,
+                    Environment.hasVariable("QAUTHMATERIALROOT") ? d`${Environment.getDirectoryValue("QAUTHMATERIALROOT")}` : undefined,

```

Comment:
>d`$ [](start = 67, length = 3)

Since getDirectoryValue returns a directory there shouldn't be a need for wrapping it in d`...`. If you do need it please file a bug on me to figure out why :)

Reply:
As Aleks suggest, I will open an other PR to address this comment. It is so difficult to get all validation finally passed.